test/integration/annotation_spec.mjs

```diff
@@ -696,4 +696,77 @@ describe("ResetForm action", () => {
       });
     });
   });
+
+  describe("Text under some annotations", () => {
+    describe("bug1885505.pdf", () => {
+      let pages;
+
+      beforeEach(async () => {
+        pages = await loadAndWait(
+          "bug1885505.pdf",
+          ":is(" +
+            [56, 58, 60, 65]
+              .map(id => `[data-annotation-id='${id}R']`)
+              .join(", ") +
+            ")"
+        );
+      });
+
+      afterEach(async () => {
+        await closePages(pages);
+      });
+
+      it("must check that an highlight as a mark element", async () => {
```

Comment:
`must check that the text under a highlight annotation exist in the DOM` should convey the intent of the test a bit more clearly (similar for the squiggly, underline and strikeout annotation types below)